code/modules/mob/living/simple_animal/hostile/handofgod.dm

```diff
@@ -0,0 +1,119 @@
+/mob/living/simple_animal/hostile/handofgod
+	name = "Hand of God"
+	desc = "A giant hand.  Smells like a great meal."
+	icon = 'icons/mob/handofgod.dmi'
+	icon_state = "handofgod"
+	icon_living = "handofgod"
+	icon_dead = "handofgod"
+	turns_per_move = 5
+	response_help_continuous = "rubs"
+	response_help_simple = "rub"
+	response_disarm_continuous = "nudges"
+	response_disarm_simple = "nudge"
+	speed = 0
+	maxHealth = 300
+	health = 300
+
+	harm_intent_damage = 0  //Mere mortal hands can't damage the visage of a god
+	obj_damage = 500  //He isn't fucking around

```

Comment:
give him the ability to smash through walls too

Reply:
ooh, and passive regen if he hasn't taken damage for a while, like what megacarps have

Reply:
I think smashing through walls kinda crosses over into the Koolaid Man's territory, and we want to keep that open for future implementation, so wall smashing is a no-go

Passive regen is fine tho